README.md

```diff
@@ -62,7 +62,7 @@ The goal is to have only projects that are mostly stable and useful to users.
 * [Games Made With Piston](https://github.com/PistonDevelopers/piston/wiki/Games-Made-With-Piston)
 * [lifthrasiir/angolmois-rust](https://github.com/lifthrasiir/angolmois-rust) — a minimalistic music video game which supports the BMS format [<img src="https://travis-ci.org/lifthrasiir/angolmois-rust.svg?branch=master">](https://travis-ci.org/lifthrasiir/angolmois-rust)
 * [swatteau/sokoban-rs](https://github.com/swatteau/sokoban-rs) — a Sokoban implementation
-* [Zone of Control](https://github.com/ozkriff/zoc) — a turn-based hexagonal strategy game [<img src="https://travis-ci.org/ozkriff/zoc.png?branch=master">](https://travis-ci.org/ozkriff/zoc)
```

Comment:
@ozkriff They looks really blurry on the Retina screens. Svg looks nice for everyone.
https://monosnap.com/file/KC6cQIZpn7WhcEqPipVcTSjCZcpH1v